### DISCUSSION TITLE
Why am I getting this shadow( in  the pic shown )

### DISCUSSION THREAD

**Discussion by Unisolgr14** [Power-Components/livewire-powergrid#630] [Q&A]
I am getting a shadow and unable to tract it .please refer the picture shown below. I also get such shadow in other parts of the table header.

![image](https://user-images.githubusercontent.com/75462488/190414492-666bce17-a96f-4cf4-a511-b191ab1dc288.png)

**Comment by akulmehta** [comment]
Are you using Bootstrap/Tailwind? What are your versions?

**Reply by Unisolgr14** [reply]
Sir I am using tailwind 3.1.8

**Reply by dansysanalyst** [reply]
Could be due to Tailwind Forms

https://livewire-powergrid.com/#/get-started/configure?id=_53-tailwind-forms

**Reply by Unisolgr14** [reply]
Thank you so much @dansysanalyst  , the shadow is gone now. Another main thing is what is the best way to set the row height  and I got a space as shown in the picture below. How to get rid of this one.
![image](https://user-images.githubusercontent.com/75462488/190546766-0ddfb715-77f6-4d10-bcf9-a04429c23c2a.png)
